executor/distsql.go

```diff
@@ -49,11 +49,13 @@ var LookupTableTaskChannelSize int32 = 50
 // lookupTableTask is created from a partial result of an index request which
 // contains the handles in those index keys.
 type lookupTableTask struct {
-	handles []int64
-	rows    []Row
-	cursor  int
-	done    bool
-	doneCh  chan error
+	// if tasksErr is not nil, this is a invalid lookupTableTask.
```

Comment:
indexHandler send lookupTableTask to a channel, then
tableHandler receive lookupTableTask from channel and further handle.

They communicate through channel, but what happens when indexHandler meet error?
it has to pass that information, so that's why there is a `tasksErr` field.
@shenli